pkg/plugin/repository/GlobalPluginRepository.go

```diff
@@ -44,6 +44,14 @@ const (
 	PLUGIN_VARIABLE_FORMAT_TYPE_DATE    PluginStepVariableFormatType = "DATE"
 )
 
+type StageType int
+
+const (
+	CI StageType = iota
```

Comment:
instead of ``iota`` we should hard coded values, because value gets changed here in reordering

Reply:
okay